src/view/edit-event-view.js

```diff
@@ -173,12 +202,27 @@ export default class EditEventView extends AbstractStatefulView {
   };
 
   #destinationChangeHandler = (evt) => {
-    const destiantion = this.#destinations.find((element) => element.name === evt.target.value);
-    this.updateElement({
-      destination: destiantion.id
+    const destination = getArrayElement(this.#destinations, evt.target.value, 'name');
+    if (destination) {
+      this.updateElement({
+        destination: destination.id
+      });
+    }
+  };
+
+  #priceChangeHandler = (evt) => {
+    this._setState({
+      basePrice: evt.target.value
     });
   };
 
+  #offerChangeHandler = (evt) => {
+    evt.preventDefault();
+    const offers = [];
+    this.element.querySelectorAll('.event__offer-checkbox').forEach((element) => (element.checked) ? offers.push(element.id) : '');
```

Comment:
Можно собрать только `checked` элементы с помощью querySelector. И сделать это в одну строчку с помощью `Array.from` (есть второй аргумент `mapFn`).

```suggestion
Array.from(this.element.querySelectorAll('.event__offer-checkbox:checked'), offer => offer.id)
```